packages/page-generator/src/storybook/FormFields.tsx

```diff
@@ -89,6 +89,14 @@ export const FormFields: (PageGeneratorField | PageGeneratorRow)[] = [
           width: '90px',
           name: 'countryCode',
         },
+        validations: [
+          {
+            message: 'Må være +47',
+            rule: (value: string) => {
+              return value === "+47";
+            },
+          },
+        ],
```

Comment:
Denne var bare for å teste nøstede validering. Er litt tullete, men det er også omg, så lar den ligge :)